src/messages/MMgrConfigure.h

```diff
@@ -35,19 +36,25 @@ class MMgrConfigure : public MessageInstance<MMgrConfigure> {
   // Default 0 means if unspecified will include all stats
   uint32_t stats_threshold = 0;
 
+  std::list<OSDPerfMetricQuery> query_list;
```

Comment:
Nit (can be addressed later): It looks like the list will be present in all MMgrConfigure messages, even those that are sent to are not osd daemons. The probably we will give a more generic name, like std::list<PerfMetricQuery>, in case we will want other daemons to provide some dynamic stats?